docs/src/surface_albedo.md

```diff
@@ -0,0 +1,40 @@
+# Surface Ocean Albedo
+
+The surface ocean albedo is the fraction of solar radiation that is reflected by the ocean surface. It is a key parameter in the Earth's energy balance and is important for understanding the ocean's role in the climate system. The surface ocean albedo is a function of the solar zenith angle, the sea surface roughness (which depends on wind speed) and wavelength of the incoming radiation.
+
+Currently, there are two methods available to specify ocean albedo:
+
+### 1) `ConstantAlbedo`:
+The default value is 0.38 and is used purely for idealized experiments.
+
+### 2) `RegressionFunctionAlbedo`:
+This is an empirically-derived parameterization of the direct and diffuse surface albedo (note, reflectivity of the inner ocean is ignored) of [Jin2011](@cite). The direct reflection is calculated using the Fresnel reflection at the air-sea interface due to the difference in refractive index between air and water. Note that the current implementation is using the broadband representation (relative refractive index, $n$, is independent of the wavelength). Its formulation is:
+
+```math
+α_{dir}(λ, μ, u) = rf(n, μ) - rf(n, μ) / rf(n0, μ) * f(μ, σ)
+```
+
+where:
+- λ is the wavelength
+- μ is the cosine of the solar zenith angle
+- u is the wind speed
+- σ is the mean wave slope distribution width
+- α(λ, μ, u) is the surface albedo
+- rf(n, μ) is the Fresnel reflectance
+- n is the relative refractive index of water and air (n = n_w/n_a)
+- n0 is the refractive index of water for visible light
```

Comment:
Should we list the parameter values here for clarity (especially where they aren't part of ClimaParameters) - these are clear from the code but not in the docs right now